hazelcast/src/main/java/com/hazelcast/map/impl/query/AggregationResultProcessor.java

```diff
@@ -39,6 +43,7 @@ public AggregationResult populateResult(Query query, long resultLimit, Collectio
 
     @Override
     public AggregationResult populateResult(Query query, long resultLimit) {
-        return new AggregationResult(query.getAggregator());
+        Aggregator resultAggregator = serializationService.toObject(serializationService.toData(query.getAggregator()));
```

Comment:
Thx @Donnerbart @jerrinot I know it's not cloned in one place, the problem is that Parallel AccumulationExecutor clones it further (one per thread) so I kept the cloning in the AccumulationExecutors. The empty result is created in the processor though, thus there's two places...